### ISSUE TITLE
Files missing 

### ISSUE THREAD

**Issue by melvin881** [matomo-org/matomo#14261]
Hello,

Your sources miss files.

Tried 2-3 methods to upload all your file (including cloning using Git), and alway got this error when trying to access index (main page for installation).

**Fatal error: Class 'Piwik\SettingsServer'**

**Comment by melvin881** [comment]
Just saw my error log, got this message.

Not sure how this message appear here, but pretty strange, i have Composer (i also installed with Git).
Anyway, will try to use build.matomo.com to see if it work.

If yes, then you should probably write here or take a look on your github files.

**Comment by melvin881** [comment]
Worked with build.

This is a real mess from Github.

**Comment by Findus23** [comment]
Hi,

The version from git is only intended for development as you need to do a few manual steps to get it running:
https://developer.matomo.org/guides/getting-started-part-1

**Comment by melvin881** [comment]
Hello,
I've used Git cloning, and didn't work anyway.

**Comment by melvin881** [comment]
Matomo is so messed up this is crazy.
I deleted my order Matomo because, API call didn't work.

Tried to update it, it crashed everything.

Now, with a fresh new install, here the error i got:

Warning: Illegal string offset 'label' in /home/xxxxxxxxx  on line 82

Warning: Illegal string offset 'nb_visits' in /home/xxxxxxxxx  on line 83
e (e hits)

Warning: Illegal string offset 'label' in /home/xxxxxxxxx  on line 82

Warning: Illegal string offset 'nb_visits' in /home/xxxxxxxxx on line 83
Y (Y hits)

I just want to know why this error is caused when the installed is new since 1 minutes lol, since i will clearly not use this unstable software.
